### PR TITLE
add page brak to learning objectives in Chemistry

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #1462

### CODE REVIEW

**Comment by ajluk** [comment]
In Chemistry and Chemistry AF Learning Objectives have different markup than other books. Additional selector in page breaks for this element is needed: 
image

File: styles/designs/common/pdf/_page-breaks.scss [hunk 1]
```diff
@@ -27,7 +27,8 @@ h1, h2, h3, h4, h5, h6,
 //Unfortunately the singular and pluaral form of these classes exists
 //LO's shouldn't be orphaned from their title 
 .learning-objectives,
-.learning-objective {
+.learning-objective,
+[data-type="page"]:not(.instroduction) > [data-type="abstract"] {
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_page-breaks.scss]: It looks like this class may be misspelled? Is this selector working: `.instroduction`

Comment (brittany-johnson) [reply styles/designs/common/pdf/_page-breaks.scss]: Ah sorry! I Just saw the screenshot. We should have this updated by content

Comment (brittany-johnson) [reply styles/designs/common/pdf/_page-breaks.scss]: Is this misspelled class `instroduction` throughout the book? @ajluk

Comment (ajluk) [reply styles/designs/common/pdf/_page-breaks.scss]: Yes it was working anyway, that's why I haven't noticed the typo. Thanks for catching it up!

**Pull request merged**